Prometheus.AspNetCore/HttpClientMetrics/HttpClientDelegatingHandlerBase.cs

```diff
@@ -68,7 +68,7 @@ protected internal TChild CreateChild(HttpRequestMessage request)
                         labelValues[i] = request.Method.Method;
                         break;
                     case HttpClientRequestLabelNames.Host:
-                        labelValues[i] = request.RequestUri.Host;
+                        labelValues[i] = request.RequestUri?.Host ?? string.Empty;
```

Comment:
is this a sensible default? opting in to nullability for net5.0 triggered a warning that `RequestUri` might be null.